common/views/components/ArchiveTree/ArchiveTree.js

```diff
@@ -770,9 +827,9 @@ const ArchiveTree = ({ work }: { work: Work }) => {
     initialLoad.current = false;
   }, [work.id]);
 
-  const TreeView = () => (
-    <Tree>
-      <TreeInstructions>{instructions}</TreeInstructions>
+  const TreeView = ({ shouldEnhance }: {| shouldEnhance: boolean |}) => (
```

Comment:
Is there a reason we're using `shouldEnhance`? That makes it sound as though it is expected to ba `false` in some occasions?

I thought the standard was to have the component look up the enhanced state using the AppContext?

Reply:
I was thinking the same thing – pick `isEnhanced` out of the `AppContext` seems like it would make more sense?